src/Microsoft.ML.Dnn/ImageClassificationTransform.cs

```diff
@@ -338,7 +338,13 @@ private IDataView GetShuffledData(string path)
 
             ImageClassificationMetrics metrics = new ImageClassificationMetrics();
             metrics.Train = new TrainMetrics();
-            for (int epoch = 0; epoch < epochs; epoch += 1)
+            //Early stopping variables
+            bool earlyStop = false;
+            int wait = 0;
+            var history = new TrainMetrics();
+            history.Accuracy = 0;
```

Comment:
Why create the a new class when you just want a variable to store accuracy? just create a float variable "lastSeenAccuracy"

Reply:
addresssed here: https://github.com/dotnet/machinelearning/pull/4237#discussion_r328833518

Reply:
addresssed here: [https://github.com/dotnet/machinelearning/pull/4237#discussion_r328833518](https://github.com/dotnet/machinelearning/pull/4237#discussion_r328833518)

---
In reply to: [328229850](https://github.com/dotnet/machinelearning/pull/4237#discussion_r328229850) [](ancestors = 328229850)